Thai charset (#2151)
manticoresoftware/manticoresearch: manual/Creating_a_table/NLP_and_tokenization/CJK.md

# Chinese, Japanese and Korean (CJK) and Thai languages

> Manticore provides built-in support for indexing languages with [continuous scripts](https://en.wikipedia.org/wiki/Scriptio_continua) (i.e., languages that does not use spaces or other marks between the words or sentences), allowing you to process texts in these languages in two different ways:

Reviewer: Manticore provides built-in support for indexing languages with [continuous scripts](https://en.wikipedia.org/wiki/Scriptio_continua) (i.e., languages that do not use spaces or other marks between words or sentences), allowing you to process texts in these languages in two different ways:

```
<!-- end -->

<!-- example cont 2 -->
2. Basic support using the N-gram options [ngram_len](../../Creating_a_table/NLP_and_tokenization/Low-level_tokenization.md#ngram_len) and [ngram_chars](../../Creating_a_table/NLP_and_tokenization/Low-level_tokenization.md#ngram_chars)

> For each language using continuous script, there are separate character set tables (`chinese`, `korean`, `japanese`, 'thai') that can be used. Alternatively, you can use the common `cont` character set table to support all CJK and Thai languages at once ot the `cjk` charset to comprise all CJK languages only. 

Reviewer: ```suggestion
For each language using a continuous script, there are separate character set tables (`chinese`, `korean`, `japanese`, `thai`) that can be used. Alternatively, you can use the common `cont` character set table to support all CJK and Thai languages at once, or the `cjk` charset to include all CJK languages only.
```
